prisma/index.ts

```diff
@@ -1,13 +1,12 @@
 import { PrismaClient } from '@prisma/client'; // Prisma ORM
 
-// add prisma to the NodeJS global type
-interface CustomNodeJsGlobal extends NodeJS.Global {
-  prisma: PrismaClient;
+// Add Prisma to the NodeJS global type
+declare global {
+  // var must be used for globally scoped variables
+  // eslint-disable-next-line no-var
+  var prisma: PrismaClient;
```

Comment:
curious why was this changes? Does it also prevent multiple instances of prisma client in development?

Reply:
Yeah good question. The most recent update to `@types/node` removed the `NodeJS.Global` type TypeScript gave us errors when extending it. Including link to SO post here: https://stackoverflow.com/a/68328402